sympy/stats/drv_types.py

```diff
@@ -834,3 +838,85 @@ def Zeta(name, s):
 
     """
     return rv(name, ZetaDistribution, s)
+
+
+#-------------------------------------------------------------------------------
+# Delaporte distribution ------------------------------------------------------------
+
+class DelaporteDistribution(SingleDiscreteDistribution):
+    _argnames = ('lamda', 'alpha', 'beta')
+    set = S.Naturals0
+
+    @staticmethod
+    def check(lamda, alpha, beta):
+        _value_check(lamda > 0, 'lambda should be positive')
```

Comment:
you spell it `lambda` here but `lamda` in the docstring; seem like docstring should be `lambda`, too. The variable name to which `lambda` is assigned is `lamda` and that's ok.

Reply:
Yeah, Rectified it now.